src/azure-cli/HISTORY.rst

```diff
@@ -23,13 +23,14 @@ Release History
 
 * GA Release Large File Shares property for storage account create and update command
 * GA Release User Delegation SAS token Support
+* Add new commands to manage blob service properties for storage account.
```

Comment:
How about we put the command group here: `az storage account blob-service-properties`

Reply:
Do we really need put so specific information here?

Reply:
Not really. We usually refer to the previous history entries.